database.cpp

```diff
@@ -3321,7 +3321,8 @@ static int sqliteLoadAllSensorsCallback(void *user, int ncols, char **colval , c
             if (!sensor.item(RStateTemperature) &&
                 sensor.modelId() != QLatin1String("lumi.sensor_switch") &&
                 !sensor.modelId().contains(QLatin1String("weather")) &&
-                !sensor.modelId().startsWith(QLatin1String("lumi.sensor_ht")))
+                !sensor.modelId().startsWith(QLatin1String("lumi.sensor_ht")) &&
+                sensor.modelId().contains(QLatin1String("86opcn01")) == false) // exclude Aqara Opple

```

Comment:
Hi, please keep the style consistent for easier reading.

`!sensor.modelId().contains(QLatin1String("86opcn01"))) // exclude Aqara Opple`

Reply:
Will do that 👍